plugins/katello/3.10/user_guide/remote_databases/index.md

```diff
@@ -176,7 +176,7 @@ systemctl start postgresql mongod
 ### Dump databases
 Dump the local databases
 ```
-foreman-maintain backup online --skip-pulp-content --preserve-directory -y /tmp/migration_backup
+katello-service backup online --skip-pulp-content --preserve-directory -y /tmp/migration_backup
```

Comment:
May want to double check that these arguments are the same in foreman-maintain - katello-service is an entirely different tool, not just a rename :)

Reply:
ah alright. Thanks @jturel

Reply:
The above arguments are present but foreman-maintain does not have a manual. So `man foreman-maintain`  fails. @jturel

Reply:
It's because you're reverting foreman-maintain to katello-service which doesn't have a backup.

Reply:
How about `foreman-maintain --help`

Reply:
> How about `foreman-maintain --help`

Yes. This works.

Reply:
> It's because you're reverting foreman-maintain to katello-service which doesn't have a backup.
I changed it back to foreman-maintain in the documentation. But I was talking about the man pages for foreman-maintain.